New-Architecture/New-Architecture-in-depth.md

```diff
@@ -390,9 +390,9 @@ The **React element tree** is used to render the **React shadow tree** in **C++*
       );
     };
 
-During the **Render phase**, a **shadow node** is created for **each React element**. This shadow node is created **synchronously**, only for React **host components**, and **not for composite components** such as **<View>**.
+During the **Render phase**, a **shadow node** is created for **each React element**. This shadow node is created **synchronously**, only for React **host components**, and **not for composite components** such as **\<View>**.
```

Comment:
Great that you noticed it. Markdown works with **\\\<View>** instead of **\<View>**. Good catch.